view/theme/frio/css/style.css

```diff
@@ -657,6 +661,22 @@ nav.navbar .nav > li > button:focus
     margin: 0px;
     padding: 12px 12px;
 }
+#search-mobile {
+    position: fixed;
+    top: 90px;
+    left: 0;
+    right: 0;
+    z-index: 2;
+
+    background-color: $background_color;
+
+    /* fix bootstrap well not playing well with data-target slide animation */
+    margin: 0;
+    padding: 0;
+    min-height: 0;
+    border-radius: 0;
+
```

Comment:
```suggestion
```

Reply:
Sorry for all the white lines, force of habit.